pontoon/insights/templates/insights/widgets/insights.html

```diff
@@ -46,65 +46,94 @@ <h3 class="controls">
+                                'name': 'Current month',
+                                'definition': 'Average age of suggestions reviewed during the specific month.',
+                            }, {
+                                'class': 'twelve-month-average',
+                                'name': '12-month average',
+                                'definition': 'Average age of suggestions reviewed in the 12 months before the specific month.',
+                            }]
+                        ) }}
+                    </h3>
+                    <canvas
+                        id="time-to-review-suggestions-chart"
+                        data-time-to-review-suggestions="{{ time_to_review_suggestions }}"
+                        data-time-to-review-suggestions-12-month-avg="{{ time_to_review_suggestions_12_month_avg }}"
+                        width="410"
+                        height="160">
+                    </canvas>
                 </div>
-                <nav class="chart-group-navigation">
-                    <ul class="unselectable">
-                        <li class="active" title="Time to review suggestions">
-                            <i class="icon"></i><span class="label">Time to review</span>
-                        </li>
-                        </li>
-                        <li title="Age of unreviewed suggestions">
-                            <i class="icon"></i><span class="label">Age of unreviewed</span>
-                        </li>
-                    </ul>
-                </nav>
             </div>
         </section>
         {% endif %}
     </section>
 
+    {% if unreviewed_lifespans %}
+    <section class="clearfix">
+        <section class="half">
+            <div class="block clearfix">
+                <div class="chart-item age-of-unreviewed-suggestions">
+                    <h3 class="controls">
+                        Age of unreviewed suggestions
+                        {{ Tooltip.display(
+                            intro='How old are the unreviewed suggestions on average at a particular point in time.',
+                            items=[{
+                                'class': 'current-month',
+                                'name': 'Current month',
+                                'definition': 'Average age of unreviewed suggestions during the specific month.',
+                            }]
+                        ) }}
+                    </h3>
+                    <canvas
+                        id="unreviewed-suggestions-lifespan-chart"
+                        data-lifespans="{{ unreviewed_lifespans }}"
+                        width="410"
+                        height="160">
+                    </canvas>
+                </div>
+            </div>
+        </section>
+        <section class="half">
+            <div class="block clearfix">
+                <div class="chart-item time-to-review-pretranslations">
```

Comment:
Why is this gated by `{% if unreviewed_lifespans %}` while `time-to-review-suggestions` isn't?

Reply:
It's gated by `unreviewed_lifespans`, because we only want to show it on the Locale Insights pages and the check is sufficient.

Are you suggesting we change it into `{% if unreviewed_lifespans and time_to_review_pretranslations %}`?

Reply:
If the current check is sufficient, there's no need to change things. Was just curious about the discrepancy.